dynamixel_controllers:
  shoulder_yaw_controller:
    controller:
      package: dynamixel_controllers
      module: joint_position_controller
      type: JointPositionController
    joint_name: shoulder_yaw
    joint_speed: 1.5
    motor:
      id: 1
      init: 512
      min: 0
      max: 1023
  shoulder_pitch_controller:
    controller:
      package: dynamixel_controllers
      module: joint_position_controller_dual_motor
      type: JointPositionControllerDual
    joint_name: shoulder_pitch
    joint_speed: 1.0
    offset: -1.57079632679
    motor_master:
      id: 2
      init: 512
      min: 200
      max: 800
    motor_slave:
      id: 3
  elbow_pitch_controller:
    controller:
      package: dynamixel_controllers
      module: joint_position_controller_dual_motor
      type: JointPositionControllerDual
    joint_name: elbow_pitch
    joint_speed: 1.0
    offset: 1.57079632679
    motor_master:
      id: 5
      init: 512
      min: 200
      max: 800
    motor_slave:
      id: 4
  wrist_pitch_controller:
    controller:
      package: dynamixel_controllers
      module: joint_position_controller
      type: JointPositionController
    joint_name: wrist_pitch
    joint_speed: 1.5
    reverse: true
    motor:
      id: 6
      init: 512
      min: 256
      max: 768
  wrist_roll_controller:
    controller:
      package: dynamixel_controllers
      module: joint_position_controller
      type: JointPositionController
    joint_name: wrist_roll
    joint_speed: 1.5
    motor:
      id: 7
      init: 512
      min: 256
      max: 768
  gripper_controller:
    controller:
      package: dynamixel_controllers
      module: joint_position_controller
      type: JointPositionController
    joint_name: finger_joint
    joint_speed: 1.5
    motor:
      id: 8
      init: 512
      min: 0
      max: 600
